src/services/utilities.ts

```diff
@@ -353,6 +353,7 @@ namespace ts {
             case SyntaxKind.MethodDeclaration:
             case SyntaxKind.MethodSignature:
                 return ScriptElementKind.memberFunctionElement;
+            case SyntaxKind.PropertyAssignment:

```

Comment:
`ShorthandPropertyAssignment` and `SpreadAssignment` look like they might be related, but I don't know the specifics of either one.

Reply:
`isVariableLike` shows a collection of related syntax kinds.

Reply:
Currently spread assignments will just turn up the binding elements in the nav bar. e.g. `var [a, b, ...other]` will show `a`, `b`, and `...other`. If we want to do better I imagine we would have to resort to the sort of merging that is done elsewhere.

checker.ts has this comment on destructuring:
```ts
        //   let { x } = obj;  // BindingElement
        //   let [ x ] = obj;  // BindingElement
        //   { x } = obj;      // ShorthandPropertyAssignment
        //   { x: v } = obj;   // PropertyAssignment
        //   [ x ] = obj;      // Expression
```
Again we would show the `BindingElement` in `let { x } = obj;` as `x`, but I'm having trouble figuring out in what case a `ShorthandPropertyAssignment` node would show up in the ` { x } = obj;` example. I just see it as separate `Block` and `ExpressionStatement` nodes.